UserManagement/migrations/0003_user_token.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.5 on 2020-10-19 22:04
```

Comment:
This might brick Django because there are now 2 files with /0003 migration files in UserManagement. I hope it doesn't do that.